esp/scm/ws_dfu.ecm

```diff
@@ -812,7 +821,43 @@ ESPresponse [exceptions_inline, nil_remove] EraseHistoryResponse
     ESParray<ESPStruct History, Origin> History;
 };
 
+ESPStruct DFUPartLocations
+{
+    int LocationIndex;
```

Comment:
@rpastrana - no problem with them remaining, am just curious why needed, since it will always be the case that array element 1, will have LocationIndex=1, array element 2 will have LocationIndex=2 etc..
i.e. it's an array where the element position is the index.